### PR TITLE
Fixed AstroidImportError

### PR DESCRIPTION
**Pull request by fadedDexofan**
Astroid raises import error because can't find `_decimal`
CPython `decimal.py` resolves import by yourself using ImportError exception handler inside.

image

### CODE REVIEW

File: pylint_django/transforms/fields.py [hunk 1]
```diff
@@ -39,11 +39,7 @@ def apply_type_shim(cls, _context=None):  # noqa
     elif cls.name == 'FloatField':
         base_nodes = scoped_nodes.builtin_lookup('float')
     elif cls.name == 'DecimalField':
-        if sys.version_info >= (3, 5):
-            # I dunno, I'm tired and this works :(
-            base_nodes = MANAGER.ast_from_module_name('_decimal').lookup('Decimal')
-        else:
-            base_nodes = MANAGER.ast_from_module_name('decimal').lookup('Decimal')
+        base_nodes = MANAGER.ast_from_module_name('decimal').lookup('Decimal')
```
Comment (atodorov) [inline pylint_django/transforms/fields.py]: Don't we have to use the same `try-except ImportError` mechanism like CPython does?

**Comment by fadedDexofan** [comment]
@atodorov check this https://github.com/PyCQA/pylint-django/issues/221

**Comment by atodorov** [comment]
@fadedDexofan please rebase so this is ready for merge.

@carlio - I'm not sure how to reliably reproduce this (short of deleting _decimal inside CI). Should I go ahead and create a separate test job where we delete `_decimal` or you are fine with merging without tests ?

**Comment by carlio** [comment]
@atodorov Yeah, looks fine to me, I mean it also gets rid of my tired hack from 3 years ago too so I'm all for it.

**Comment by atodorov** [comment]
@fadedDexofan can you `git rebase -i master` instead of merging master back into your own branch ?

**Pull request merged**